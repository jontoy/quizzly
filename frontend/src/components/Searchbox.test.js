import React from "react";
import { render } from "@testing-library/react";
import Searchbox from "./Searchbox";

it("renders without crashing", () => {
  render(<Searchbox />);
});

it("matches snapshot", () => {
  const { asFragment } = render(<Searchbox />);
  expect(asFragment()).toMatchSnapshot();
});
