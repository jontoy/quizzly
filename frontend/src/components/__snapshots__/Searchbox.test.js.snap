// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <form
    class="Searchbox form-inline mt-3"
  >
    <input
      class="form-control form-control-lg flex-grow-1"
      name="search"
      placeholder="Search..."
      type="search"
      value=""
    />
  </form>
</DocumentFragment>
`;
